### PR TITLE
loglevel

### PR DESCRIPTION
**Pull request by NilsG-S**
Adds `loglevel` to the project.

@asclines In `logger.js` I have an if-statement that checks `typeof ENV !== 'undefined'`. This is because I wasn't sure how you wanted to set the build type. I have Webpack setup to replace `ENV` with `prod`, `dev`, or `test` accordingly, but I couldn't see anything similar on the back end.

### CODE REVIEW

**Comment by asclines** [comment]
@NilsG-S to my knowledge there is nowhere in the backend that really checks that.

The only occurrences of using environment variables in the API are [here](https://github.com/TTUSDC/cpceed-webapp/blob/master/server.js#L11)
and [here](https://github.com/TTUSDC/cpceed-webapp/blob/master/server.js#L18)

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Time logging app for the CPCEED program at Texas Tech University",
   "main": "src/index.jsx",
   "scripts": {
-    "start": "npm-run-all start-api start-app",
+    "start": "npm-run-all build start-api",
```
Comment (asclines) [inline package.json]: Good catch

File: webpack/dev.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const dev = webpackMerge(base, {
   entry: './src/index.jsx',
 
   // Enables source maps that can be accessed in browser dev tools
-  devtool: 'source-map',
+  devtool: 'cheap-module-eval-source-map',
```
Comment (asclines) [inline webpack/dev.js]: For my understanding, what is this?

Comment (NilsG-S) [inline webpack/dev.js]: Source maps are used to translate the lines of the bundle created by Webpack to lines in the individual files that comprise it. This change reduces the cost of creating the source maps.

**Pull request merged**